### DISCUSSION TITLE
blog/creating-a-brainrot-language-server-in-golang/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#23] [Q&A]
# blog/creating-a-brainrot-language-server-in-golang/

I was reading the language server protocol docs and about json-rpc for collaborating on a new project which will need an LSP server. Since it was getting a little boring to just read so much of documentations and RFCs, I thought of making a small and fun project around LSPs.
🌐 You can find the source code of this project in this github repository What’s an LSP? The Language Server Protocol (LSP) is a game-changer in how we build development tools.

https://jitesh117.github.io/blog/creating-a-brainrot-language-server-in-golang/

**Comment by harish876** [comment]
lmaoooo, this is sick

**Reply by Jitesh117** [reply]
Thanks man!
